presidio-analyzer/presidio_analyzer/recognizer_registry.py

```diff
@@ -236,7 +244,8 @@ def get_recognizers(
         )
 
         if not to_return:
-            raise ValueError("No matching recognizers were found to serve the request.")
+            raise ValueError("No matching recognizers were found to serve the\
```

Comment:
Looks like this lines fails the e2e tests FYI

Reply:
Oh! Thank you @RoeyBC. I passed flake8 and recommended this change...

I'll try again.